diagnostics/seaice_suite/seaice_suite_sic_mean_sigma.py

```diff
@@ -0,0 +1,365 @@
+# For a one-year lag, the map for January shows the correlation of January
+# and January a year later. And so forth.
+#
+# The results from this POD are to be appear in a paper being prepared
+# for the MAPP team special issue.
+# 
+#   Last update: 1/25/2021
+# 
+#   - Version/revision information: version 1 (1/31/2021)
+#   - PI Cecilia Bitz, University of Washington bitz@uw.edu
+#   - Developer/point Lettie Roach and Cecilia Bitz
+#   - Lettie Roach, University of Washington, lroach@uw.edu
+# 
+#   Open source copyright agreement
+# 
+#   The MDTF framework is distributed under the LGPLv3 license (see LICENSE.txt). 
+#   Unless you've distirbuted your script elsewhere, you don't need to change this.
+# 
+#   Functionality
+# 
+#   Code to input many years of monthly sea ice concentration and compute spatial maps of
+#     monthly climatology, monthly standard deviation of all years
+#     linear trend of each month for all years
+#     monthly detrended standard deviation for all years
+#     lagged correlation by month of monthly detrended data (hence also deseasonalized)
+#       where lag is an arbitrary number of months
+# 
+#   Required programming language and libraries
+# 
+#      Python3 
+# 
+#   Required model output variables
+# 
+#      Sea ice concentration either as fraciton or percent 
+#      it is assumed to be in the sea ice realm, and normally is named siconce
+#      minor edits can alter to the settings.jsconc file can change these assumptions
+# 
+#   References
+# 
+#      Roach, L.R. and Co-authors, 2021: Process-oriented evaluation of Sea Ice 
+#         and Mixed Layer Depth in MDTF Special Issue
+#
+
+#from __future__ import print_function
+import os
+import matplotlib
+matplotlib.use('Agg') # non-X windows backend
+
+import xarray as xr                # python library we use to read netcdf files
+import matplotlib.pyplot as plt    # python library we use to make plots
+import xesmf as xe
+import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+
+# homegrown this code also imports pandas and scipy
+from seaice_MLD_stats import (
+    xr_reshape,
+    _lrm,
+    _lagcorr,
+)
+
+
+
+def readindata(file, varname='siconc',firstyr='1979',lastyr='2014'):
+    ds = xr.open_dataset(file)
+    ds = ds.sel(time=slice(firstyr+'-01-01',lastyr+'-12-31')) # limit to yrs of interest, maybe model dep
+    print('Limit domain to Arctic to match obs') # script would work fine if data were global
+    ds = ds.where(ds.latitude>30.,drop=True) # limit to arctic for now, remove later
+    field = ds[varname]
+    field.name = varname
+
+    if field.isel(time=0).max().values > 50.0:  # ensure we are [0,1] not [0,100]
+        field=field*0.01
+
+    ds.close()
+    return field
+
+
+
+
+### 1) Loading model data files: ###############################################
+
+input_file = "{DATADIR}/mon/{CASENAME}.{siconc_var}.mon.nc".format(**os.environ)
```

Comment:
These definitions should be removed: absolute paths to all requested variables are set via the `path_variable` setting in the new settings file format, which defaults to `<variable name>_FILE`; see [documentation](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_settings.html#varlist-entry-properties). Calls to open_dataset(), etc. below should also be changed.